### ISSUE TITLE
Check version of `psr/log` as it fails on install

### ISSUE THREAD

**Issue by jacotri77** [scoutapp/scout-apm-php#53]
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - scoutapp/scout-apm-laravel v0.1.1 requires scoutapp/scout-apm-php 0.1.1 -> satisfiable by scoutapp/scout-apm-php[v0.1.1].
    - Installation request for scoutapp/scout-apm-laravel ^0.1.1 -> satisfiable by scoutapp/scout-apm-laravel[v0.1.1].
    - Conclusion: remove psr/log 1.0.2
    - Conclusion: don't install psr/log 1.0.2
    - scoutapp/scout-apm-php v0.1.1 requires psr/log ^1.1 -> satisfiable by psr/log[1.1.0, 1.1.x-dev].
    - Can only install one of: psr/log[1.1.0, 1.0.2].
    - Can only install one of: psr/log[1.1.x-dev, 1.0.2].
    - Installation request for psr/log (locked at 1.0.2) -> satisfiable by psr/log[1.0.2].


Installation failed, reverting ./composer.json to its original content.
```

**Comment by asgrim** [comment]
This has been fixed in `dev-master` - for now you could use that as a workaround until the next release.

 @cschneid is it worth doing a minor release (new features such as PHP ext integration are in, so it's not a patch release really, even though it's pre-release) now for this?

**Comment by cschneid** [comment]
Yeah, probably worth releasing in the next few days

**Comment by cschneid** [comment]
Released in 0.2.0. We now only require psr/log 1.0 and up.
